maintenance.md

```diff
@@ -952,12 +946,15 @@ TBD
 - csi-snapshotter
 
 1. Check the [release page](https://github.com/rook/rook/releases).
-2. Check the upstream Dockerfile. If there are any updates, update our `Dockerfile`.
+2. Check the `CEPH_VERSION` of the upstream Makefile. If the version is changed, update the base image version in our `Dockerfile`.
```

Comment:
I think we used the ceph even if its version is different from the one in the makefile when the version of it is supported.

Reply:
In my understanding, it seems that at least the major version of Ceph was implicitly aligned with the upstream Makefile of Rook. My proposal is that we should write the policy in the maintenance document (and should not use the older majour version of Ceph even if that is supported by Rook).

Moreover, I believe we should use the exact same version of Ceph as the one in Rook's upstream Makefile. If we don't, there's no guarantee that those version combinations will work properly.